### DISCUSSION TITLE
What is the procedure in place now?

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#18] [Q&A] [answered]
1. Receive Hard Copy Paper Work
2. Someone manually inputs data
3. ??Followup with pet owner stating any issues with paperwork??
4. Pet Arrives at Airport
5. Pet is transported to Animal Quarantine
6. Animal is Inspected
7. Animal is ready to be picked up?

**Comment by judyn808** [comment]
This is referring to Direct Airport release procedure where the owner mails in the documents in advance?
I have heard some people getting emailed that their paperwork was not in order, but I do not know if that is the norm or exception.

For this particular process, I'll note that a "card" is printed in advance so that when the owner arrives, AQS is already set with their paperwork. I'm not sure when the card is created though (before or after pet arrives). Is this important enough to ask Chris?

**Reply by JJ-Foust** [reply]
I actually don't think this matters at all anymore. They don't want integration. This was my brainstorming when I was thinking they wanted to optimize the whole process. So i was trying to think about times during the process we could integrate. But they want a stand alone system that does not integrate.

**Comment by JJ-Foust** [comment, marked as answer]
No longer matter since we are not integrating.
